src/encoded/types/dataset.py

```diff
@@ -1386,6 +1386,86 @@ class FunctionalCharacterizationSeries(Series):
         'elements_references.files',
     ]
 
+    @calculated_property(condition='related_datasets', schema={
+        "title": "Examined loci",
+        "type": "array",
+        "items": {
+            "type": "object",
+        },
+        "notSubmittable": True,
+    })
+    def examined_loci(self, request, related_datasets=None):
```

Comment:
Would something like `return request.select_distinct_values('examined_loci', *related_datasets)` work the same way?

Reply:
nope it fails

Reply:
Probably need a concrete field that could be deduplicated. Looks like `examined_loci` doesn't have `@id` or anything.